scripts/travis/build.sh

```diff
@@ -27,7 +27,11 @@ if [[ "${WITH_DISTRIBUTED}" = "mpi" ]]; then
     export OMPI_CXX=${CXX}
     CC="mpicc"
     CXX="mpicxx"
+    if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
```

Comment:
The else branch hasn't been indented.
An alternative way to do this is to "build up" the `launch` variable like this:
```bash
launch="mpiexec -n 4"
if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
    launch="$launch --oversubscribe --mca btl tcp,self"
fi
```

Reply:
Because the flags are not standard mpirun flags, it would be useful to have a small comment that explains why the `--oversubscribe` and `--mca` are used on os x.